Dockerfile.agent

```
@@ -1,19 +1,23 @@
-# Install the required packages
-RUN pip install --no-cache-dir -r requirements.txt
+# Security-conscious organizations should package/review uv themselves.
+COPY --from=ghcr.io/astral-sh/uv:latest /uv /usr/local/bin/uv
 
 # Copy the application code
 COPY . .
 
-# Expose the port if your application uses one
-# EXPOSE <port_number>
+# set up a virtual env to use for whatever app is destined for this container.
+RUN uv venv --python 3.12.5 /venv
```

Hadolint

[error] Ensure the shebang uses an absolute path to the interpreter.

(SC2239)